spdb/spatialdb/spatialdb.py

```diff
@@ -789,9 +790,15 @@ def write_cuboid(self, resource, corner, resolution, cuboid_data, time_sample_st
         # Get keys ready
         experiment = resource.get_experiment()
         if iso is True and resolution > resource.get_isotropic_level() and experiment.hierarchy_method.lower() == "anisotropic":
-            base_write_cuboid_key = "WRITE-CUBOID&ISO&{}&{}".format(resource.get_lookup_key(), resolution)
+            if to_black:
+                base_write_cuboid_key = "BLACK-CUBOID&ISO&{}&{}".format(resource.get_lookup_key(), resolution)
```

Comment:
I think there will be updates necessary to `rediskvio.py` to make it check for this new prefix. The cache manager code (in `boss-tools.git/cachemgr/boss_delayedwrite.py`) uses code in that class to send the cuboid writes to the s3 flush lambda.